core/cmd/local_client_test.go

```diff
@@ -339,7 +339,11 @@ func TestClient_RunNodeWithAPICredentialsFile(t *testing.T) {
 			c := cli.NewContext(nil, set, nil)
 
 			if test.wantError {
-				assert.EqualError(t, client.RunNode(c), "error creating api initializer: open doesntexist.txt: no such file or directory")
+				err = client.RunNode(c)
+				assert.Error(t, err)
+				if err != nil {
```

Comment:
IIRC you could simplify here:
```suggestion
				if assert.Error(t, err) {
```